it/data/SampleData.scala

```diff
@@ -299,4 +299,25 @@ trait SampleData {
     )
   )
 
+  val updateCaseJson: JsObject = Json.obj(
+    "caseId" -> "C18",
+    "additionalInfo" -> "Additional Info",
+    "supportingDocumentTypes" -> Json.arr(
```

Comment:
we don't have supportDocTypes in the update case interface.

Reply:
removed it :+1: